netbench-orchestrator/src/russula/netbench.rs

```diff
@@ -73,6 +73,7 @@ impl ServerContext {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
why all the dead code?

Reply:
The orchestrator contains 2 [bins](https://github.com/aws/s2n-netbench/blob/main/netbench-orchestrator/Cargo.toml#L11-L17) so clippy complains about the ones we are using for `russula_cli`

Reply:
ok, I think there is a way to address this by putting all the `mod`s in a `lib.rs` instead of the bin, but we don't need to block on this. Can you open an issue to address this though, its a little ugly with so much not-dead dead code

Reply:
https://github.com/aws/s2n-netbench/issues/38